### PR TITLE
Add more keywords

### PR DESCRIPTION
**Pull request by iwanttodo1t**
Add more keywords

### CODE REVIEW

File: src/util/chat.hpp [hunk 1]
```diff
@@ -51,7 +53,6 @@ namespace
 	    "<b>",
 	    "P888",
 	    "gtacash",
-	    ".cc",
```
Comment (Rxann) [inline src/util/chat.hpp:49]: Add this back and replace it with .co

**Comment by sch-lda** [comment]
Although I haven't tested it in yimmenu, is this as expected?
image
image

**Comment by iwanttodo1t** [comment]
I am a newbie in C++, sorry for the trouble.

**Comment by Rxann** [comment]
> I am a newbie in C++, sorry for the trouble.

It is fine, I was there and still partially am now.

**Comment by iwanttodo1t** [comment]
Some ads use special characters instead of regular letters
For example:  с（Russian alphabet：eg .сo）

（ΑΒ）ΓΔ（ΕΖΗ）Θ（ΙΚ）Λ（ΜΝ）Ξ（Ο）ΠΡΣ（ΤΥ）Φ（Χ）ΨΩαβγδεζνξοπρσηθικλμτυφχψω
（А）Б（В）ГДЕЁЖЗИЙКЛ（МНО）П（РСТ）УФХЦЧШЩЪЫЬЭЮЯабвгдеёжзийклмнопрстуфхцчшщъыьэюя
āáǎàōóǒòēéěèīíǐìūúǔùǖǘǚǜüê（ɑ）ńň（ɡ）

File: src/util/chat.hpp [hunk 2]
```diff
@@ -25,42 +25,75 @@ namespace
...
-
+	//
 	static const char* spam_texts[] = {
-	    "qq", //a chinese chat app
 	    "www.",
 	    ".cn",
 	    ".cc",
 	    ".com",
 	    ".top",
+	    ".co",
```
Comment (lonelybud) [inline src/util/chat.hpp:35]: looks very aggressive and others as well. Imagine your friend typing a name of website to tell you. He will be flagged as spammer by someone using yim. Its a very rare case though

Comment (ghost) [reply src/util/chat.hpp:35]: > looks very aggressive and others as well. Imagine your friend typing a name of website to tell you.

I guess that's what trusted players are for?

**Comment by lonelybud** [comment]
I also want to comment that timer based detection which is disabled by default is very aggressive. I have seen modders copying and pasting text and sending messages and they can be detected as spammer by timer based detections if enabled by yim user. The solution for that is to detect how many times the user is doing that paste & send (say 3 times) and if the interval between message arrivals from that user are very precise then its fine to flag them.

**Pull request closed without merging**